addons/api/addon/serializers/managed-group.js

```diff
@@ -7,4 +8,29 @@ export default class ManagedGroupSerializer extends ApplicationSerializer {
    * @type {boolean}
    */
   serializeScopeID = false;
+
+  serialize() {
+    const serialized = super.serialize(...arguments);
+
+    if (serialized.attributes.filter_string) {
+      serialized.attributes.filter = serialized.attributes.filter_string;
+    } else {
+      serialized.attributes.filter = null;
```

Comment:
Is this needed? There's a test called `it serializes correctly on create with no filter provided` that explicitly tests that filter is set to `null` when it's not present, I actually couldn't find where it was getting set to null so wasn't sure of the intention.

Reply:
By default, empty strings are serialised to `null`.  On serialisation, you might want to check that `filter_string` exists as a key on `attributes` (or check that it is strictly not `undefined`), rather than simply falsy.  That should eliminate the need for an else case, I believe?

Reply:
That makes sense, I'll update it